$NetBSD: patch-configure.ac,v 1.2 2016/02/12 23:42:15 jperkin Exp $

Use the installed libabiword instead of the one in the build directory
This is needed for the plugins so the can build as a separate package

Disable Cocoa configuration on Darwin, it doesn't build.

--- configure.ac.orig	2014-12-21 04:51:43.000000000 +0000
+++ configure.ac
@@ -146,7 +146,7 @@ fi
 TOOLKIT_IS_GTK2="no"
 AC_MSG_CHECKING([for platform and toolkit])
 case ${host_os} in
-  *darwin*)
+  *notdarwin*)
     PLATFORM="unix"
     TOOLKIT="cocoa"
     ;;
@@ -893,7 +893,7 @@ if test "$TOOLKIT" = "gtk"  && 
    test "$enable_shared" = "yes"; then
 	# link plugins to work around gcc visibility issue with 
 	# derived classes in dlopened modules
-	PLUGIN_LIBS='${top_builddir}/src/libabiword-'"$ABIWORD_SERIES"'.la'" $PLUGIN_LIBS"
+	PLUGIN_LIBS='${prefix}/lib/libabiword-'"$ABIWORD_SERIES"'.la'" $PLUGIN_LIBS"
 elif test "$TOOLKIT" = "win"; then
 	PLUGIN_CFLAGS="$PLUGIN_CFLAGS "'-D_WIN32_IE=0x0501 -D_WIN32_WINNT=0x0500'
 	PLUGIN_LIBS='${top_builddir}/src/libabiword-'"$ABIWORD_SERIES"'.la'" $PLUGIN_LIBS"
